runtime/shibuya/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "shibuya-runtime"
-version = "5.15.0"
+version = "5.16.0"
```

Comment:
One thing I missed before - we always strive(d) to keep all semver's the same for client, local runtime, shibuya, shiden and astar runtimes.

I don't mind we change how we do this, but let's keep it consistent for now.
We can discuss it next Tuesday perhaps.

Reply:
Yeah, I do want to bring up the semver and spec version management for discussion. Let's do it in the next sync then. Will update to follow the current rule.